resources/js/components/User/mixins/InteractWithUser.js

```diff
@@ -27,8 +27,7 @@ export default {
                     }
                 })
                 .catch((error) => {
-                    console.error(error)
```

Comment:
Do we no longer want to log these errors?

Reply:
[They already get logged with the notification](https://github.com/rapidez/core/blob/master/resources/js/package.js#L135) so this was just doubled up